components/Radio/src/Radio.cpp

```diff
@@ -11,58 +11,109 @@
 
 #include "Radio.h"
 
+#include "freertos/queue.h"
 #include <driver/uart.h>
 #include <esp_log.h>
+#include <string.h>
+
+
+
+#define FTDI_PORT UART_NUM_0
```

Comment:
We can't use `UART_NUM_0` while testing cause it's used by the FTDI and for the stdout (serial console), use `UART_NUM_1` or `UART_NUM_2`. Can the `#define` be moved to `defaults.h`?

Reply:
yes, the port can be moved and the definition as well